libs/partners/astradb/langchain_astradb/utils/astradb.py

```diff
@@ -51,20 +52,29 @@ def __init__(
             )
 
         if astra_db:
-            self.astra_db = astra_db
+            self.astra_db = astra_db.copy()
             if async_astra_db:
-                self.async_astra_db = async_astra_db
+                self.async_astra_db = async_astra_db.copy()
             else:
                 self.async_astra_db = self.astra_db.to_async()
         elif async_astra_db:
-            self.async_astra_db = async_astra_db
+            self.async_astra_db = async_astra_db.copy()
             self.astra_db = self.async_astra_db.to_sync()
         else:
             raise ValueError(
                 "Must provide 'astra_db_client' or 'async_astra_db_client' or "
                 "'token' and 'api_endpoint'"
             )
 
+        self.astra_db.set_caller(
+            caller_name="langchain",
+            caller_version=getattr(langchain_core, "__version__", None),
```

Comment:
You should get the version of `langchain_astradb`. Can also get langchain_core if helpful for debugging, but I think the partner version is much more important

Reply:
Right, thank you for the suggestion. But we'd prefer to keep alignment between the caller_name being "langchain", as in the core Lc machinery, and the corresponding version.
In other words, we might eventually extend this to a list of (package,version) pairs - in which case we'd neatly add `("langchain_astradb", <corresponding version>)` to the stack.

So if that's ok we'd like to start by leaving it as is.

Reply:
@eyurtsev So I would say this is good to go as far as this is concerned :)

Reply:
Not the caller name, but the caller version. I don't think you care about the version of langchain core but of langchain_astradb? 

I'll merge but consider changing the version that you're using